diff --git a/node_modules/next/navigation-types/compat/navigation.d.ts b/node_modules/next/navigation-types/compat/navigation.d.ts
index 6a1f925..d90d67c 100644
--- a/node_modules/next/navigation-types/compat/navigation.d.ts
+++ b/node_modules/next/navigation-types/compat/navigation.d.ts
@@ -8,7 +8,7 @@ declare module 'next/navigation' {
    * If used from `pages/`, the hook may return `null` when the router is not
    * ready.
    */
-  export function useSearchParams(): ReadonlyURLSearchParams | null
+  function useSearchParams(): ReadonlyURLSearchParams | null
 
   /**
    * Get the current pathname. For example, if the URL is
@@ -17,7 +17,7 @@ declare module 'next/navigation' {
    * If the hook is accessed from `pages/`, the pathname may be `null` when the
    * router is not ready.
    */
-  export function usePathname(): string | null
+  function usePathname(): string | null
 
   /**
    * Get the current parameters. For example useParams() on /dashboard/[team]
@@ -25,7 +25,7 @@ declare module 'next/navigation' {
    *
    * If used from `pages/`, the hook will return `null`.
    */
-  export function useParams<
+  function useParams<
     T extends Record<string, string | string[]> = Record<
       string,
       string | string[]
@@ -38,7 +38,7 @@ declare module 'next/navigation' {
    *
    * If used from `pages/`, the hook will return `null`.
    */
-  export function useSelectedLayoutSegments(): string[] | null
+  function useSelectedLayoutSegments(): string[] | null
 
   /**
    * A [Client Component](https://nextjs.org/docs/app/building-your-application/rendering/client-components) hook
@@ -46,5 +46,5 @@ declare module 'next/navigation' {
    *
    * If used from `pages/`, the hook will return `null`.
    */
-  export function useSelectedLayoutSegment(): string | null
+  function useSelectedLayoutSegment(): string | null
 }
